components/api/src/main/java/com/hotels/styx/api/StyxCoreObservable.java

```diff
@@ -91,12 +91,9 @@ public StyxCoreObservable(CompletionStage<T> future) {
         return fromSingleObservable(delegate);
     }
 
-    private static <T> CompletableFuture<T> fromSingleObservable(Observable<T> observable) {
+    public static <T> CompletableFuture<T> fromSingleObservable(Observable<T> observable) {
```

Comment:
Do we need this method to be public now? It takes an Observable and returns a CompletableFuture, so the class doesn't seem to be the right place for this method.

Reply:
My silly mistake. Fixed.